### PR TITLE
ci: build helm charts

### PR DESCRIPTION
**Pull request by KFearsoff**
Needs #6 merged first

### CODE REVIEW

**Review by DovnarAlexander — APPROVED** [review]

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -15,15 +15,25 @@ jobs:
       TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   helm-release:
-    runs-on: ubuntu-latest
-    steps:
-      - run: echo "placeholder"
+    uses: naviteq/github-actions/.github/workflows/create-helm-release.yaml@main
```
Comment (DovnarAlexander) [inline .github/workflows/main.yaml]: Let's add dependency for docker build.

**Review by DovnarAlexander — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -15,15 +15,25 @@ jobs:
...
-    steps:
-      - run: echo "placeholder"
     needs: [helm-release]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+      - name: Build
+        run: |
```
Comment (DovnarAlexander) [inline .github/workflows/main.yaml]: Let's include that in our action instead

**Review by DovnarAlexander — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml (hunk 2 above)
Comment (KFearsoff) [reply .github/workflows/main.yaml]: I assume we'd need to create a separate action in https://github.com/naviteq/github-actions ?

Comment (KFearsoff) [reply .github/workflows/main.yaml]: Created it at
https://github.com/naviteq/github-actions/pull/16
Tests pass: 
https://github.com/naviteq/curator-opensearch/actions/runs/3685945374

**Review by DovnarAlexander — APPROVED** [review]

**Pull request merged**